examples/hallowelt/go_app/README.md

```diff
@@ -0,0 +1,253 @@
+```
+We also need to change the code that invoked the smart contract
+```bash
+    # store the value of 100 in asset1
+    say "- invoke storeAsset transaction to store value 100 in asset1"
+    try ${PEER_CMD} chaincode invoke -o ${ORDERER_ADDR} -C ${CHAN_ID} -n ${CC_ID} -c '{"Args":["storeAsset","asset1","100"]}' --waitForEvent
+
+    # retrieve current value for "asset1";  should be 100;
+    say "- invoke retrieveAsset transaction to retrieve current value of asset1"
+    try ${PEER_CMD} chaincode invoke -o ${ORDERER_ADDR} -C ${CHAN_ID} -n ${CC_ID} -c '{"Args":["retrieveAsset","asset1"]}' --waitForEvent
+
+    #say "- invoke query with retrieveAsset transaction to retrieve current value of asset1"
+    try ${PEER_CMD} chaincode query  -o ${ORDERER_ADDR} -C ${CHAN_ID} -n ${CC_ID} -c '{"Args":["retrieveAsset","asset1"]}'
+```
+We will replace it with an invocation to the go program we just created.
+```bash
+	go run .
+```
+Just as simple as that! The bash script should end up looking as the following:
+```bash
+#!/bin/bash
+
+SCRIPTDIR="$(dirname $(readlink --canonicalize ${BASH_SOURCE}))"
+FPC_PATH="${SCRIPTDIR}/../.."
+FABRIC_CFG_PATH="${SCRIPTDIR}/../../integration/config"
+FABRIC_SCRIPTDIR="${FPC_PATH}/fabric/bin/"
+
+. ${FABRIC_SCRIPTDIR}/lib/common_utils.sh
+. ${FABRIC_SCRIPTDIR}/lib/common_ledger.sh
+
+#this is the path that will be used for the docker build of the chaincode enclave
+CC_PATH=${FPC_PATH}/examples/helloworld/_build/lib/
+
+CC_ID=helloworld_test
+CC_VER="$(cat ${CC_PATH}/mrenclave)"
+CC_EP="OR('SampleOrg.member')"
+CC_SEQ="1"
+
+helloworld_test() {
+    say "- do hello world"
+
+    # install helloworld  chaincode
+    # input:  CC_ID:chaincode name; CC_VER:chaincode version;
+    #         CC_PATH:path to build artifacts
+    say "- install helloworld chaincode"
+    PKG=/tmp/${CC_ID}.tar.gz
+    try ${PEER_CMD} lifecycle chaincode package --lang fpc-c --label ${CC_ID} --path ${CC_PATH} ${PKG}
+    try ${PEER_CMD} lifecycle chaincode install ${PKG}
+
+    PKG_ID=$(${PEER_CMD} lifecycle chaincode queryinstalled | awk "/Package ID: ${CC_ID}/{print}" | sed -n 's/^Package ID: //; s/, Label:.*$//;p')
+
+    try ${PEER_CMD} lifecycle chaincode approveformyorg -o ${ORDERER_ADDR} -C ${CHAN_ID} --package-id ${PKG_ID} --name ${CC_ID} --version ${CC_VER} --sequence ${CC_SEQ} --signature-policy ${CC_EP}
+    try ${PEER_CMD} lifecycle chaincode checkcommitreadiness -C ${CHAN_ID} --name ${CC_ID} --version ${CC_VER} --sequence ${CC_SEQ} --signature-policy ${CC_EP}
+    try ${PEER_CMD} lifecycle chaincode commit -o ${ORDERER_ADDR} -C ${CHAN_ID} --name ${CC_ID} --version ${CC_VER} --sequence ${CC_SEQ} --signature-policy ${CC_EP}
+
+    # create an FPC Chaincode enclave
+    try ${PEER_CMD} lifecycle chaincode initEnclave -o ${ORDERER_ADDR} --peerAddresses "localhost:7051" --name ${CC_ID}
+
+    say "- invoking FPC chaincode using go client SDK"
+    export CC_ID
+    GOAPP="./go_app"
+    go run $GOAPP
+#    go run .
```

Comment:
this is not match the description above. :/

Reply:
oh! forgot to change there! thanks!